docs/installing/dashboard.rst

```diff
@@ -46,7 +46,7 @@ Now, lets create the dashboard application:
 
 ::
 
-    tsuru app create tsuru-dashboard python -t admin
+    tsuru app create tsuru-dashboard python --team admin
```

Comment:
This is not required and is not the point of the tutorial. If the user is part of only one team there's no need to specify the `--team` flag. We could add this observation in the docs I'm not opposed to that, but only adding this unnecessary flag during a tutorial of `app create` doesn't clarify anything in my opinion.